assets/app/views/directives/osc-key-values.html

```diff
@@ -7,10 +7,12 @@
                  name="key"
                  ng-attr-placeholder="{{keyTitle}}"
                  ng-model="key"
+                 ng-model-options="{ debounce: 100 }"
```

Comment:
Yeah but i am tinkering with it a bit more.  removing.  we aren't showing an error message if it matches, just disabling.  seems to be sufficient enough indication that you already have used this key name.  

We could also show "already used key" but in this case I tend to think it would be distracting/overkill.  thoughts?